AstSemantics.md

```diff
@@ -530,3 +530,5 @@ In the MVP, `has_feature` would always return false. As features were added post
 `has_feature` would start returning true. `has_feature` is a pure function, always
 returning the same value for the same string over the lifetime of a single
 instance and other related (as defined by the host environment) instances.
```

Comment:
What I mean is that we can't `#ifdef` out a function if it has a SIMD type in its signature. If SIMD isn't supported then none of the `.wasm` file can use it in a signature. If `has_feature` were instead a structural container for sub-s-expressions (including top-level ones) then this would be possible.